src/VisualStudio/Core/Impl/Options/AbstractOptionPageControl.cs

```diff
@@ -202,6 +222,35 @@ internal virtual void OnSave()
         internal virtual void Close()
         {
         }
+
+        internal virtual void OnSearch(string searchString)
+        {
+            var shouldScrollIntoView = true;
+            foreach (var handler in _searchHandlers)
+            {
+                if (handler.TryHighlightSearchString(searchString) && shouldScrollIntoView)
+                {
+                    handler.EnsureVisible();
+                    shouldScrollIntoView = false;
+                }
+            }
+        }
+
+        private protected void AddSearchHandler(ComboBox comboBox)
+        {
+            foreach (ComboBoxItem item in comboBox.Items)
+            {
+                AddSearchHandler(item);
+            }
+        }
+
+        private protected void AddSearchHandler(ContentControl control)
+        {
+            if (control.Content is string content)
```

Comment:
We should probably add a debug assert or something here instead of doing nothing.